microraiden/microraiden/config.py

```diff
@@ -66,7 +66,7 @@
 
 # required version of the deployed contract at CHANNEL_MANAGER_ADDRESS.
 # Proxy will refuse to start if the versions do not match.
-MICRORAIDEN_VERSION = "0.1.0"
+MICRORAIDEN_VERSION = "0.1.1"
```

Comment:
The next release version will be `0.2.0`. Why the version change now?

Reply:
I've accidentally pushed package without javascript library to the pypi and to fix it I had to bump maintenance version to be able to re-upload the package. Now I realize that string from `config.py` has to be equal to contract version.

Reply:
We should probably compare only major+minor version of the contract and package, otherwise it'll be painful to do any maintenance updates.

Reply:
agreed.